### ISSUE TITLE
Blank window on osx 

### ISSUE THREAD

**Issue by vlabo** [maps4print/azul#35]
## Description
I followed the instructions of the tutorial and build the app for osx, also tied the exampels in the repository. But every time I see a blank window. I then tried added a button and loged on every click the loged  appied but the button is invisible.

The same code works on Windows machine that I tried. Dose enyone hava an idea were is the problem. I can try to fix it by myself if someone directs me were to look.

## Version / OS

* Operating system:
MacOS Majave version 10.14

**Comment by ralfbiedert** [comment]
I think this is a duplicate of #5.

**Comment by vlabo** [comment]
Yeah I read it sounds similar but I can't get any of the examples to work even when I folow the instructions they gave

**Comment by fschutt** [comment]
Okay, this is a bit hard to follow for me:

- What is the code you tried to run? If it's just an example, it can't be that long, so just paste what you tried to run, where you just saw a blank window.
- What are you seeing when running the examples (screenshots would be nice)?
- Did you get any errors, whether compilation or runtime errors?

Notice that a "blank window" is the expected behaviour for the hello-world tutorial. If you add a button and then still nothing happened, well that would be a bug, but an empty `Dom::new(NodeType::Div)` will just give you an empty window.

**Comment by fschutt** [comment]
@vlabo Oh, yeah, on OSX, I misread that, sorry. Yes, OSX is currently very much under construction. We've already tried debugging it twice, with the only result being that it has something to do with webrender not redrawing correctly, even though all the events get handled. The only problem is that I don't have a mac so I can currently not debug mac-related issues.

**Comment by Yatekii** [comment]
Guess I need to debug on mach a little further then. As if I didn't check all the places yet ... but webrender really seems to work properly, so it must be an azul issue I feel like. Or at least with winit.

**Comment by vlabo** [comment]
I will try to debug it the next few days and will post if I find something

**Comment by cetra3** [comment]
This is what I see trying to run the calculator app:

![](https://user-images.githubusercontent.com/6415435/47683593-f65a2580-dc1f-11e8-932a-734899daf8b0.png)

I'm running OSX 10.14 Mojave

*Note: I am seeing the same blank screen with webrender examples.*

**Comment by vlabo** [comment]
That makes sense, something normal for mac I guess. At least we now were is the problem now. Thanks

**Comment by cetra3** [comment]
@fschutt Yep, that's what I assumed was happening, hence why I tested both azul & webrender.
